test/bigtest/tests/override-item-block-test.js

```diff
@@ -113,20 +113,16 @@ describe('Override item block', () => {
         beforeEach(async function () {
           await checkOut.errorModal.overrideButton.click();
 
-          this.server.post('/circulation/check-out-by-barcode', (schema, request) => {
-            return (
-              {
-                'userId': user.id,
-                'itemId': item.id,
-                'status': {
-                  'name': 'Open'
-                },
-                'loanDate': '2017-03-05T18:32:31Z',
-                'action': 'checkedOutThroughOverride',
-                'loanPolicyId': loanPolicyWithLimitId,
-                item
-              }
-            );
+          this.server.post('/circulation/check-out-by-barcode', {
```

Comment:
Could you please remove unnecessary ' below?

Reply:
Done. Looks cleaner now, thanks